source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -25,6 +26,16 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::HeaderMap& headers, bool)
 
   state_ = Calling;
   stopped_ = false;
+
+  // Per the spec
+  // http://www.w3.org/TR/cors/#cross-origin-request-with-preflight-0, CORS
+  // pre-flight requests shouldn't include user credentials.
+  if (headers.Method() &&
```

Comment:
I think this bypass too much request by blindly accepting all OPTIONS method, OPTIONS are not only used for CORS, it need additional check or guarded by configuration.

Reply:
Do you have a recommendation for additional check?

Reply:
check all three header existence? https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request

Reply:
btw I think this can be achieved with config change and deosn't require code change, no? @qiwzhang